easybuild/easyconfigs/b/BLAST+/BLAST+-2.2.30-goolf-1.4.10.eb

```diff
@@ -0,0 +1,42 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+#
+# Copyright:: Copyright 2012-2013 University of Luxembourg/Luxembourg Centre for Systems Biomedicine
+# Authors::   Fotis Georgatos <fotis.georgatos@uni.lu>, Kenneth Hoste (UGent)
+# License::   MIT/GPL
+# $Id$
+#
+# This work implements a part of the HPCBIOS project and is a component of the policy:
+# http://hpcbios.readthedocs.org/en/latest/HPCBIOS_2012-94.html
+##
+
+easyblock = 'ConfigureMake'
+
+name = 'BLAST+'
+version = "2.2.30"
+
+homepage = 'http://blast.ncbi.nlm.nih.gov/'
+description = """Basic Local Alignment Search Tool, or BLAST, is an algorithm
+ for comparing primary biological sequence information, such as the amino-acid
+ sequences of different proteins or the nucleotides of DNA sequences."""
+
+toolchain = {'name': 'goolf', 'version': '1.4.10'}
+
+sources = ['ncbi-blast-%(version)s+-src.tar.gz']
+source_urls = ['http://ftp.ncbi.nlm.nih.gov/blast/executables/blast+/%(version)s/']
+
+# patch not needed in BLAST+ 2.2.31 or higher
+# http://www.ncbi.nlm.nih.gov/viewvc/v1?view=revision&revision=65204
+patches = ['blast+_%(version)s.patch']
+
+dependencies = [('Boost', '1.51.0')]
```

Comment:
like @JensTimmerman  mentioned, any reason to keep this at 1.51?

sorry to keep coming up with new remarks @pescobar :)